packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -813,6 +813,11 @@ describe("typespec-client-generator-core: package", () => {
       strictEqual(methodResponse.kind, "method");
       strictEqual(methodResponse.type, widgetModel);
       strictEqual(createOrUpdate.response.resultPath, "result");
+      strictEqual(createOrUpdate.response.resultSegments?.length, 1);
```

Comment:
emitter side may need to know if this is property/parameter tagged with `@continuationToken` then do some special handling, how do we know that?

Reply:
this info is in method, not in response.